product/strategy.md

```diff
@@ -0,0 +1,108 @@
+
+### FlowForge for enterprise
+
+The cornerstone of FlowForge is to make Node-RED more accessible to enterprises. Therefore, we are constantly implementing requirements that support the easy use of Node-RED in productive environments. We will focus on:
+
+1. High availability & Scalability
+
+2. State-of-the-art built-in security e.g. VPN support
+
+3. DevOps
+   
+   1. Change control
+   
+   2. Collaborative Development
+   
+   3. Pipelines (e.g. staging)
+
+### Node-RED at edge
+
+Many organizations position Node-RED instances on remote servers, edge or industrial devices, to meet network, infrastructure, or use case requirements. Therefore, management of remote instances is crucial for the overall success of closing the gap between IT and OT. 
+
+FlowForge already supports this process by creating snapshots on Node-RED instances that can be deployed to multiple remote targets. We will continue to expand the functionalities to allow users to manage existing Node-RED instances, improve their development practices to accelerate development and ensure a reliable and repeatable process, providing more flexible remote deployment options and offering the best support for devices in closed and segmented networks as we are facing it in productive environments.
+
+### Certified nodes marketplace
+
+The "Node-RED Library" offers a unique added value to many Node-RED and FlowForge users. It is already possible today to easily create technical implementations out of the box for several OT and IT protocols, thanks to the community and Node-RED library. 
+
+The logical consequence of FlowForge’s journey is to also include “custom nodes” as one element of the product. Therefore, FlowForge will offer official support for certain popular nodes and in addition enable the community to offer commercial “nodes” via FlowForge. Nevertheless, an open-source core is always required and wanted. However, premium extensions and commercial support allow the community to participate in the success of FlowForge.
+
+### IIoT Solutions
+
+Today, a whole range of use cases can be implemented with Node-RED. From KPI calculations to intralogistics solutions, incredible use cases are already being realized in Node-RED. 
+
+To make it easy for FlowForge users in particular to get started and address their specific need, a use case library is to be created (based on the "Node-RED Library"), which offers standard templates for specific IT and OT related use cases. This way, we as FlowForge address direct added values in production, for example:
+
+1. Overall Equipment Effectiveness (OEE) calculation and visualization
+
+2. ANDON for Node-RED
+
+3. Process monitoring for tightening equipment
+
+4. Predictive maintenance analysis
```

Comment:
Predictive maintenance is closely aligned with OEE, maybe we should include these together?

Reply:
I'd argue these are different solutions?

Reply:
Yes, different solutions. 

OEE is a retrospective KPI that measures how "well" or "effectively" the overall production, lines, or even individual machines have produced, based on several factors. 

Predictive Maintenance means to forecast the breakdown of individual components of a machine (and so the machine/equipment itself) before a respective event occurs. As a result, it can have a positive impact on the availability of the machine and consequently on the OEE.

However, the way to calculate OEE has nothing to do with the approach to predicting maintenance events.